jekyll/_cci2/server/v4.1/operator/using-metrics.adoc

```diff
@@ -29,68 +29,14 @@ Your CircleCI server installation collects a number of metrics and logs by defau
 [#telegraf]
 === Telegraf
 Most services running on server report StatsD metrics to the https://www.influxdata.com/time-series-platform/telegraf/[Telegraf] pod running in server.
-The configuration is fully customizable, so you can forward your metrics from Telegraf to any output supported by Telegraf through https://docs.influxdata.com/telegraf/v1.17/plugins/#output-plugins[output plugins]. By default, it will provide a metrics endpoint for Prometheus to scrape.
+The configuration is fully customizable, so you can forward your metrics from Telegraf to any output supported by Telegraf through https://docs.influxdata.com/telegraf/v1.24/plugins/#output-plugins[output plugins]. By default, it will provide a metrics endpoint for Prometheus to scrape.
```

Comment:
@circleci/security-controls / Server reviewers:

The (somewhat) big change to the way Telegraf is now being configured (which we don't state anywhere explicitly) is that we are _technically_ no longer allowing the end user to completely replace the default telegraf configuration. This has to do with the changes to [telegraf-config.yaml](https://github.com/circleci/server/blob/main/templates/telegraf-config.yaml) in that the custom configuration is now getting _appended_ to what we ship by [default](https://github.com/circleci/server/blob/a77fd029851c0ccb5752b8d2ca5aae0e9f3c0419/values.yaml#L513-L552).The reason for this has to do with the metrics/usage statistics work where we needed to have sections of Telegraf config that the user shouldn't be able to override (otherwise we would have lost our metrics shipping config).

The **TL;DR** to the above is that an en user can adjust any aspect of Telegraf config except for the default `statsd` input and `prometheus` output plugin config sections. Anything else can be overwritten by suppling Telegraf compatible config yaml in `telgraf.config.custom_config_file`.

Reply:
I haven't added a note to the documentation about the above ☝️ because I don't think there's a point and it might introduce unnecessary confusion...